packages/ruleset/src/functions/operation-id-naming-convention.js

```diff
@@ -2,23 +2,12 @@ const pickBy = require('lodash/pickBy');
 const reduce = require('lodash/reduce');
 const merge = require('lodash/merge');
 const each = require('lodash/each');
+const { operationMethods } = require('../utils');
```

Comment:
Moved the common "operationMethods" variable to be a constant in the "utils" module.